python/pyspark/ml/feature.py

```diff
@@ -512,14 +512,19 @@ class HashingTF(JavaTransformer, HasInputCol, HasOutputCol, HasNumFeatures, Java
     .. versionadded:: 1.3.0
     """
 
+    binary = Param(Params._dummy(), "binary", "If true, all non zero counts are set to 1. " +
+                   "This is useful for discrete probabilistic models that model binary events " +
+                   "rather than integer counts. (default: False)",
```

Comment:
The style seems to be `. Default False` rather than `. (default: False)`. @BryanCutler @holdenk thoughts?

Though I must say I'd prefer `(default: X).` across the board myself.